src/Illuminate/Process/FakeProcessResult.php

```diff
@@ -135,6 +136,14 @@ public function output()
         return $this->output;
     }
 
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
We don't typically use `@inheritdoc` from an interface. We does use it when extending for example Symfony's classes etc

Reply:
Ok good to know will change that.